### PR TITLE
Add ytmdesktop v1.11.0

### PR DESCRIPTION
**Pull request by adlerluiz**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] There are no [open pull requests](https://github.com/Homebrew/homebrew-cask/pulls) for the same update.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).

## note
For token reference the app have 'desktop' and 'app' on name.

### CODE REVIEW

File: Casks/ytmdesktop.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'ytmdesktop' do
+  version '1.11.0'
+  sha256 '2a6a0280d4242d5a0ae48e021999269dac8b5d755115ef2d0d09b521947d61e4'
+
+  url "https://dl.ytmdesktop.app/mac/YouTube%20Music%20Desktop%20App-#{version}.dmg"
+  appcast 'https://ytmdesktop.app/#download'
```
Comment (vitorgalvao) [inline Casks/ytmdesktop.rb:6]: `https://github.com/ytmdesktop/ytmdesktop/releases.atom`

File: Casks/ytmdesktop.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+cask 'ytmdesktop' do
```
Comment (vitorgalvao) [inline Casks/ytmdesktop.rb:1]: Either `adlerluiz-youtube-music` or `ytmdesktop-youtube-music`. `ytmdesktop` ignores the token reference and `youtube-music` looks deceptively official.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/ytmdesktop.rb (hunk 2 above)
Comment (adlerluiz) [reply Casks/ytmdesktop.rb:1]: maybe ```ytmdesktop-app``` or ```ytmd-app``` or ```ytmd-player```

File: Casks/ytmdesktop.rb (hunk 1 above)
Comment (adlerluiz) [reply Casks/ytmdesktop.rb:6]: I will modify, thank you

**Comment by adlerluiz** [comment]
The build has already finished successfully, but here's a bug
image

https://travis-ci.com/github/Homebrew/homebrew-cask/builds/172113020

File: Casks/ytmdesktop.rb (hunk 2 above)
Comment (vitorgalvao) [reply Casks/ytmdesktop.rb]: > maybe `ytmdesktop-app` or `ytmd-app` or `ytmd-player`

No. Specifically one of the two I mentioned. Do not ignore the token reference. Cask naming has clear guidelines.

**Review by adlerluiz — COMMENTED** [review]
changed

File: Casks/ytmdesktop.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+  version '1.11.0'
+  sha256 '2a6a0280d4242d5a0ae48e021999269dac8b5d755115ef2d0d09b521947d61e4'
+
+  url "https://dl.ytmdesktop.app/mac/YouTube%20Music%20Desktop%20App-#{version}.dmg"
+  appcast 'https://ytmdesktop.app/#download'
+  name 'YouTube Music Desktop App'
+  homepage 'https://ytmdesktop.app/'
+
+  app 'YouTube Music Desktop App.app'
+end
```
Comment (ran-dall) [inline Casks/ytmdesktop.rb]:
```suggestion
  zap trash: [
               '~/Library/Preferences/app.ytmd.plist',
               '~/Library/Saved Application State/app.ytmd.savedState',
             ]
end
```

**Review by ran-dall — CHANGES_REQUESTED** [review]
@adlerluiz You were also missing the `zap` staza.

**Review by ran-dall — CHANGES_REQUESTED** [review]
@adlerluiz Your 'style' was a big off. You have some extra spaces.

File: Casks/ytmdesktop-youtube-music.rb [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+  name 'YouTube Music Desktop App'
+  homepage 'https://ytmdesktop.app/'
+
+  app 'YouTube Music Desktop App.app'
+  
+  zap trash: [
+               '~/Library/Preferences/app.ytmd.plist',
+               '~/Library/Saved Application State/app.ytmd.savedState',
+             ]
+end
```
Comment (ran-dall) [inline Casks/ytmdesktop-youtube-music.rb:16]:
```suggestion

  zap trash: [
               '~/Library/Preferences/app.ytmd.plist',
               '~/Library/Saved Application State/app.ytmd.savedState',
             ]
end
```

**Review by ran-dall — CHANGES_REQUESTED** [review]

File: Casks/ytmdesktop-youtube-music.rb (hunk 4 above)
Comment (adlerluiz) [reply Casks/ytmdesktop-youtube-music.rb]: I hope that everything is now right =)

**Review by ran-dall — APPROVED** [review]

**Comment by ran-dall** [comment]
@adlerluiz Everything LGTM

**Comment by adlerluiz** [comment]
```Done. Your build exited with 0.```

thanks for the support

**Comment by adlerluiz** [comment]
I believe that there is something wrong with the hook, because in travis it is like ok and here still queued.

https://travis-ci.com/github/Homebrew/homebrew-cask/jobs/352935240

**Comment by ran-dall** [comment]
@adlerluiz It's all good. The CIs go bananas all the time. It's something I'm working on. I'll simply leaving this open overnight if @vitorgalvao want's to take a look and merge; otherwise, I'll merge it for you tomorrow morning.

**Comment by adlerluiz** [comment]
great, thank you

**Pull request merged**